tensorflow/core/kernels/sparse/mat_mul_op.cc

```diff
@@ -728,48 +728,42 @@ namespace {
 template <typename T>
 struct GPUDataType;
 
+// GPUDataType templates are currently not instantiated in the ROCm flow
+// So leaving out the #elif TENSORFLOW_USE_ROCM blocks for now
+// hipblas library is not (yet) being pulled in via rocm_configure.bzl
+// so cannot reference tyeps from hipblas headers here
 template <>
 struct GPUDataType<Eigen::half> {
 #if GOOGLE_CUDA
```

Comment:
Nit: Can we wrap `#if GOOGLE_CUDA` around the whole struct GPUDataType definitions (lines 728-774)? It looks cleaner.
Or we can leave it like this if you plan to add hipblas soon.

Reply:
Never mind, @av8ramit and I decided to just pull this in now. :)